### PR TITLE
Adding version 5.1.1 version history

### PR DESCRIPTION
**Pull request by hflynn**
Based on draft text from https://trello.com/c/oac6QylW/318-5-1-1-omero-history-txt

### CODE REVIEW

File: history.txt [hunk 1]
```diff
@@ -1,6 +1,33 @@
...
+For OMERO.insight:
+
+-  improved open actions
+-  tidying of the menu structure
+-  correction of the mouse zoom behavior
+-  fix for the Drag-n-Drop functionality
+
+Other updates include:
+
+-  overhaul of the CLI session login logic
```
Comment (mtbc) [inline history.txt]: maybe "log-in"; just a thought

File: history.txt [hunk 2]
```diff
@@ -1,6 +1,33 @@
...
+-  cleaning and testing of all code examples
+-  further removal of the use of deprecated methods, including the complete
+   migration to Graph2
```
Comment (mtbc) [inline history.txt]: maybe, "... to the `GraphModify2` graph requests"; we don't actually have anything named Graph2. (@ximenesuk: complete for 5.1.1?)

**Comment by joshmoore** [comment]
Merging as a perfectly rationale explanation of things. If we've forgotten anything, a last minute PR early next week would be very much doable. Thanks all!

**Comment by joshmoore** [comment]
...*sigh* but of course, some things need to be removed. Sorry for the noise! Should have left well-enough alone on a Friday EOB.

**Comment by hflynn** [comment]
--no-rebase

**Comment by hflynn** [comment]
What do you need me to remove @joshmoore ?

**Comment by joshmoore** [comment]
`including the complete migration to ``GraphModify2`` graph requests` cc: @ximenesuk

**Comment by hflynn** [comment]
Ok, will open a new PR now

**Pull request merged**